uxarray/helpers.py

```diff
@@ -434,3 +434,33 @@ def _is_ugrid(ds):
         return True
     else:
         return False
+
+
+def grid_center_lat_lon(ds):
```

Comment:
When Anissa and I were working on this part of the SCRIP writer function, we felt it would be a good function that users could access outside of just being for creating the grid_center_lat/lon variables in SCRIP files. There is also an implication from the ESMF gist I was looking at that implies having this as a stand alone function would be helpful in both creating ESMF and SCRIP style datasets. That said, I am more than happy to put it into the scrip file and have it as an internal API function instead. Whatever is the preference from everyone else is completely fine by me as well!

Reply:
Ok, that sounds reasonable! Then, please document its existence in user API accordingly